sqlparser/analyzer_test.go

```diff
@@ -477,6 +477,33 @@ func TestExtractSetValues(t *testing.T) {
 		sql:   "set session sql_safe_updates = 0",
 		out:   map[SetKey]interface{}{{Key: "sql_safe_updates", Scope: "session"}: int64(0)},
 		scope: "session",
+	}, {
+		sql: "set search_path to 'value'",
+		out: map[SetKey]interface{}{{Key: "search_path", Scope: "session"}: "value"},
```

Comment:
I saw a comment for `ExtractSetValues` function that `SET` supports strings integers and booleans. can we check them too?